src/code42cli/errors.py

```diff
@@ -124,6 +105,14 @@ def invoke(self, ctx):
         except click.exceptions.Exit:
             raise
 
+        except (
+            UserDoesNotExistError,
+            Py42UserAlreadyAddedError,
```

Comment:
What about the LegalHold-related errors?

Reply:
It doesn't appear to be necessary for some reason

Reply:
Oh wait, one of them is, will add

Reply:
There was only one unique err for legalhold